comp/logs/agent/flare/flare_controller.go

```diff
@@ -79,6 +114,31 @@ func (fc *FlareController) SetAllFiles(files []string) {
 	fc.allFiles = files
 }
 
+// SetEnabledSDSRules stores the names of the enabled SDS rules received
+// through RC.
+func (fc *FlareController) SetEnabledSDSRules(rulesNames []string) {
+	fc.mu.Lock()
+	defer fc.mu.Unlock()
+
+	fc.sdsInfos.enabledRulesNames = append([]string{}, rulesNames...)
```

Comment:
The function says "set", but this line uses `append`. Is this expected? Does this list grow indefinitely each time SDS is reconfigured?

Reply:
It is appending to an empty slice (I'm copying the slice just to make sure I'm good)